src/components/privacyPolicy/PrivacyPolicy.js

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Container from '../common/Container';
+import { StyledPrivacyPolicy, StyledPrivacyPolicyContainer } from './style';
+
+const PrivacyPolicy = () => (
+  <Container>
+    <StyledPrivacyPolicy>
+      <StyledPrivacyPolicyContainer>
+        <h2> PRIVACY POLICY FOR SAY THEIR NAMES</h2>
+        <p>
+          At Say Their Names, accessible from
+          {' '}
+          <Link to="/"> saytheirnames.io</Link>
+          , one of our main priorities is
+          the privacy of our visitors. This Privacy Policy document contains
+          types of information that is collected and recorded by Say Their Names
+          and how we use it.
+        </p>
+
+        <p>
+          If you have additional questions or require more information about our
+          Privacy Policy, do not hesitate to contact us.
+        </p>
+
+        <p>
+          This Privacy Policy applies only to our online activities and is valid
+          for visitors to our  and mobile apps with regards to the information that they
```

Comment:
Missing website here. 
```suggestion
          for visitors to our  and mobile apps with regards to the information that they
```